test/quantization/pass/test_insert_quantize_on_dtype_mismatch.py

```diff
@@ -107,38 +117,92 @@ def setup(self, mod: torch.nn.Module, target_op, scale=1.0, zp=0, dtype="uint8")
 
     def run_test(self):
         # Before pass
-        self.assertNotEqual(self.target.meta[QPARAM_KEY].dtype, self.dtype)
+        # InsertQuantizeOnDtypeMismatch works for the case when input dtype does not match with output dtype
```

Comment:
```suggestion
        # InsertQuantizeOnDtypeMismatchTest works for the case when input dtype does not match with output dtype
```

(optional) The pass `InsertQuantizeOnDtypeMismatch` works on all cases, it's `InsertQuantizeOnDtypeMismatchTest` that works for mismatch case. So to be the comment technically right.